ldes-server-infra-postgres/postgres-pagination-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/pagination/postgres/repository/PageMemberEntityRepository.java

```diff
@@ -39,4 +39,6 @@ select v.name, count(*)
 		  group by v.name
 		""", nativeQuery = true)
 	List<Tuple> getPaginatedMemberCounts(String collectionName);
+
+	void deleteAllByBucket_View_EventStream_NameAndBucket_View_NameAndMember_IdIn(String collectionName, String viewName, List<Long> memberIds);
```

Comment:
I am not a huge fan of such a long method name with underscores and etc, but I know the is required by JPA. In such cases, I would personally write the query myself, but this is more a personal opinion than a AP